io.github.idevecore.Valuta.json

```
@@ -0,0 +1,61 @@
+{
+  "app-id": "io.github.idevecore.Valuta",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "46",
+  "sdk": "org.gnome.Sdk",
+  "command": "valuta",
+  "finish-args": [
+    "--share=network",
+    "--share=ipc",
+    "--device=dri",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--own-name=io.github.idevecore.Valuta.SearchProvider"
+  ],
+  "cleanup": [
+    "*blueprint*"
+  ],
```

```suggestion
```

---

Actually this:

```suggestion
  "cleanup": [
    "*"
  ],
```

It's even in the doc.

---

I made the required change.